....................................FF.....F.F......FFFFF...........
======================================================================
FAIL: runTest (util.TestRst_1_Document5_FullRstDemoLossyRst-MpeWriterTestCase)
TestRst_1_Document5_FullRstDemo; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 204, in __test_writer
    writer=writer)['whole']#writer_name='dotmpe-rst')
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/core.py", line 448, in publish_parts
    enable_exit_status=enable_exit_status)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/core.py", line 662, in publish_programmatically
    output = pub.publish(enable_exit_status=enable_exit_status)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/core.py", line 219, in publish
    output = self.writer.write(self.document, self.destination)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/writers/__init__.py", line 80, in write
    self.translate()
  File "/Volumes/Simza/conf-mpe/lib/py/dotmpe/du/ext/writer/rst.py", line 42, in translate
    self.document.walkabout(visitor)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 166, in walkabout
    visitor.dispatch_visit(self)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/nodes.py", line 1882, in dispatch_visit
    return method(node)
  File "/Volumes/Simza/conf-mpe/lib/py/dotmpe/du/ext/writer/rst.py", line 510, in visit_subtitle
    assert not in_sidebar, "TODO"
AssertionError: TODO

======================================================================
FAIL: runTest (util.TestRst_1_Document6LossyRst-MpeWriterTestCase)
TestRst_1_Document6; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.1.document-6.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.1.document-6.rst:8: (WARNING/2) Bibliographic field "Authors" incompatible with extraction: it must contain either a single paragraph (with authors separated by one of ";,"), multiple paragraphs (one per author), or a bullet list with one paragraph (one author) per item.
/Volumes/Simza/project/docutils-ext/var/test-rst.1.document-6.rst:: (ERROR/3) Unknown target name: "goodger@python.org".


======================================================================
FAIL: runTest (util.TestRst_22_DocinfoLossyRst-MpeWriterTestCase)
TestRst_22_Docinfo; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.22.docinfo.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.22.docinfo.rst:8: (WARNING/2) Bibliographic field "Authors" incompatible with extraction: it must contain either a single paragraph (with authors separated by one of ";,"), multiple paragraphs (one per author), or a bullet list with one paragraph (one author) per item.
/Volumes/Simza/project/docutils-ext/var/test-rst.22.docinfo.rst:: (ERROR/3) Unknown target name: "goodger@python.org".


======================================================================
FAIL: runTest (util.TestRst_24_ReferencesLossyRst-MpeWriterTestCase)
TestRst_24_References; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:20: (WARNING/2) malformed hyperlink target.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:22: (WARNING/2) Inline target start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:31: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:50: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:58: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:65: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:71: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:77: (WARNING/2) malformed hyperlink target.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:79: (ERROR/3) Content block expected for the "replace" directive; none found.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:79: (WARNING/2) Substitution definition "EXAMPLE" empty or invalid.

.. |EXAMPLE| replace::

/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:84: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:88: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:96: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:100: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:116: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:122: (ERROR/3) Anonymous hyperlink mismatch: 0 references but 1 targets.
See "backrefs" attribute for IDs.
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:3: (ERROR/3) Unknown target name: "example-ref-name-1".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:39: (ERROR/3) Unknown target name: "example-name-5".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:39: (ERROR/3) Unknown target name: "example-name-5".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:39: (ERROR/3) Unknown target name: "example-name-5".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:39: (ERROR/3) Unknown target name: "python".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:52: (ERROR/3) Unknown target name: "test".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:52: (ERROR/3) Unknown target name: "refs".
/Volumes/Simza/project/docutils-ext/var/test-rst.24.references.rst:57: (ERROR/3) Unknown target name: "refs".


======================================================================
FAIL: runTest (util.TestRst_5_Inline3LossyRst-MpeWriterTestCase)
TestRst_5_Inline3; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-3.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-3.rst:10: (ERROR/3) Unknown interpreted text role "role".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-3.rst:12: (ERROR/3) Unknown interpreted text role "role".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-3.rst:14: (ERROR/3) Unknown interpreted text role "role".


======================================================================
FAIL: runTest (util.TestRst_5_Inline4LossyRst-MpeWriterTestCase)
TestRst_5_Inline4; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:13: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:16: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:40: (WARNING/2) malformed hyperlink target.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:42: (ERROR/3) Content block expected for the "replace" directive; none found.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:42: (WARNING/2) Substitution definition "EXAMPLE" empty or invalid.

.. |EXAMPLE| replace::

/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Too many symbol footnote references: only 0 corresponding footnotes available.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "python".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "example".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "site".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:19: (ERROR/3) Unknown target name: "pep 287".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:19: (ERROR/3) Unknown target name: "rfc 2822".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-4.rst:58: (ERROR/3) Unknown target name: "label".


======================================================================
FAIL: runTest (util.TestRst_5_Inline5LossyRst-MpeWriterTestCase)
TestRst_5_Inline5; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-5.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-5.rst:10: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-5.rst:12: (ERROR/3) Anonymous hyperlink mismatch: 0 references but 1 targets.
See "backrefs" attribute for IDs.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-5.rst:1: (ERROR/3) Unknown target name: "reference".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-5.rst:1: (ERROR/3) Unknown target name: "reference".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-5.rst:6: (ERROR/3) Unknown target name: "reference".


======================================================================
FAIL: runTest (util.TestRst_5_Inline6LossyRst-MpeWriterTestCase)
TestRst_5_Inline6; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:3: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:5: (ERROR/3) Content block expected for the "parsed-literal" directive; none found.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:12: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:16: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:18: (ERROR/3) Anonymous hyperlink mismatch: 0 references but 2 targets.
See "backrefs" attribute for IDs.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:8: (ERROR/3) Unknown target name: "ref".
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-6.rst:8: (ERROR/3) Unknown target name: "ref".


======================================================================
FAIL: runTest (util.TestRst_5_Inline7LossyRst-MpeWriterTestCase)
TestRst_5_Inline7; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 253, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 158, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/Volumes/Simza/project/docutils-ext/test/util.py", line 228, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-7.rst>

/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-7.rst:6: (WARNING/2) malformed hyperlink target.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-7.rst:9: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.
/Volumes/Simza/project/docutils-ext/var/test-rst.5.inline-7.rst:2: (ERROR/3) Unknown target name: "term".


----------------------------------------------------------------------
Ran 68 tests in 2.485s

FAILED (failures=9)
